components/Navbar.tsx

```diff
@@ -104,115 +204,144 @@ export const Navbar: React.FC = () => {
-                size="icon"
-                onClick={handleSearchToggle}
-              ></Button>
-            )}
+                ></Button>
+              )}
+            </div>
           </div>
         </div>
-      </div>
-    </nav>
+      </nav>
+    </header>
   )
 }
 
-const NavigationLinks: React.FC<NavigationProps> = ({
-  routes,
-  parentTitle,
-  parentHref,
-}) => {
+const NavigationSectionContent: React.FC<{
+  groups: RouteGroup[];
+  parentTitle: string;
+}> = ({ groups }) => {
+  const hasMultipleGroups = groups.length > 1;
+
   return (
-    <NavigationMenu.Content className="absolute top-[100%] z-50 w-max rounded-md shadow-md">
-      <ul className="list-none p-4 m-0 bg-neutral-50 flex flex-col gap-2">
-        <li className="p-2 hover:bg-white hover:text-black">
-          <NavigationMenu.Link
-            href={parentHref}
-            className="p-2 focus:outline-none focus:text-black"
-          >
-            Explore {parentTitle}
-          </NavigationMenu.Link>
-        </li>
-        {routes.length > 0 &&
-          routes.map((route) => (
-            <li key={route.href} className="p-2  hover:text-black">
-              <NavigationMenu.Link
-                href={route.href}
-                className="p-2 focus:outline-none focus:text-black"
-              >
-                {route.name}
-              </NavigationMenu.Link>
-            </li>
-          ))}
-      </ul>
+    <NavigationMenu.Content className="absolute top-full z-50 w-max rounded-md shadow-md bg-white">
+      <div className={`p-10 flex ${hasMultipleGroups ? 'space-x-8' : ''} ${hasMultipleGroups ? 'flex-grow' : ''}`}>
+        {groups.map((group, index) => (
+          <div
+          key={group.name}
+          className={`${hasMultipleGroups && index > 0 ? 'border-l border-black pl-10' : ''} ${hasMultipleGroups ? 'flex-1' : ''}`}
+        >
+            {group.name && <h3 className="text-2xl pl-10 mb-6 tracking-widest">{group.name.toUpperCase()}</h3>}
+            <ul className="list-none pl-10 flex flex-col gap-2">
+              {group.routes.map((route) => (
+                <li key={route.href} className="hover:bg-white hover:text-black pb-4">
+                  <NavigationMenu.Link
+                    href={route.href}
+                    className="flex items-start focus:outline-none focus:text-black"
+                  >
+                    {route.icon && (
+                      <div className="mr-4 h-[4rem] w-[4rem] bg-neutral-100/75 border text-white rounded-md flex items-center justify-center">
+                        <route.icon className="h-1/2 w-1/2"/>
+                      </div>
+                    )}
+                    <div className="flex flex-col">
+                      <span className="text-2xl font-semibold pb-1">{route.name}</span>
+                      {route.description && (
+                        <span className="text-lg text-neutral-500 italic mb-3">{route.description}</span>
```

Comment:
```suggestion
                        <span className="max-w-[550px] text-lg text-neutral-500 italic mb-3">{route.description}</span>
```
The description of project consulting was too long and forcing the icon to shrink its width. Adding a max width to the descipriton text seems to solve the issue.